doc/macsec/MACsec_hld.md

```diff
@@ -673,7 +693,6 @@ MACsec SECY Table
     "InPktsUnknownSCI":{{InPktsUnknownSCI}} # the number of the received frame with unknown SCI
     "InPktsNoSCI":{{InPktsNoSCI}}           # the number of the received frame without SCI (those frames will be passed to uncontrolled port)
     "InPktsOverrun":{{InPktsOverrun}}       # the number of the received frame that was discarded because the validation capabilities of the Cipher Suite cannot support current rate
-
```

Comment:
If pfc packets are received before macsec session is established, what is the behavior? if they are dropped,  where are they accounted?

Reply:
We have an ACL entry for dropping all data traffic before macsec session is established from the sender side. And to this ACL entry, it treats the PFC as same as normal data frames, so we don't know how many PFC were dropped by this ACL entry before macsec session is established.
But if this counter is needed, I believe we can have a special ACL entry for dropping the PFC frames before macsec session is established. Is it required or just the icing on the cake?